domains/certificates/Query.API/API/ContractService/Clients/MeteringPointsClient.cs

```diff
@@ -36,15 +36,18 @@ public MeteringPointsClient(HttpClient httpClient, IHttpContextAccessor httpCont
         ValidateOwnerAndSubjectMatch(owner);
         SetApiVersionHeader();
 
-        var meteringPointsUrl = IsBearerTokenIssuedByB2C() ? $"/api/measurements/meteringpoints?organizationId={owner}" : "/api/measurements/meteringpoints";
+        var meteringPointsUrl = IsBearerTokenIssuedByB2C()
```

Comment:
"/api/measurements/meteringpoints" should we boyscout delete if statement since all is AzureB2C now?

Reply:
Renamed